test/new_relic/agent/configuration/default_source_test.rb

```diff
@@ -140,12 +140,6 @@ def test_config_search_path_in_warbler
       end
     end
 
-    def test_application_logging_enabled_default
```

Comment:
This test conflicts with intention of allowing certain values for Booleans, and defaulting to the default value when an unrecognized value is given. This test is saying if `:foo` is set as the value for `application_logging.enabled`, we should use it. Ripping it out!